estolowka-backend/src/main/java/pl/prodzajto/estolowkabackend/order/Day.java

```diff
@@ -4,25 +4,29 @@
 import lombok.Builder;
 import lombok.Data;
 import lombok.NoArgsConstructor;
+import pl.prodzajto.estolowkabackend.order.validator.NotAllMealsFalse;
 
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.validation.constraints.NotNull;
 import java.time.LocalDate;
 
 @Entity
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
 @Builder
-class Day {
+@NotAllMealsFalse
+public class Day {
 
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private boolean breakfast;
     private boolean dinner;
     private boolean supper;
+    @NotNull
```

Comment:
I think it should be validated in DTO object that comes from client side, not in entity layer.